src/index.js

```diff
@@ -1,4 +1,4 @@
-import lolex from 'lolex'
+import * as fakeTimers from '@sinonjs/fake-timers'
```

Comment:
Instead of `import * as`, let's just import `install`. Maybe this is an over-optimization, but if we only use `install` and there are other exports we don't use, this will be more tree-shakable.

Reply:
That's a good idea, I've updated it.